Hundreds of "SyntaxWarning: invalid escape sequence" will break in future python versions

### Describe the bug

We see many syntax warnings with python 3.12 which will be syntax errors in a future versions of python.

### Related command

Any command that goes through the bad code, including `az login`.

### Errors

```
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/batch/models/_models_py3.py:4756: SyntaxWarning: invalid escape sequence '\s'
  """A Job Preparation Task to run before any Tasks of the Job on any given
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/mgmt/servicelinker/models/_models_py3.py:63: SyntaxWarning: invalid escape sequence '\ '
  """The access key directly from target resource properties, which target service is Azure Resource, such as Microsoft.Storage.
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/mgmt/servicelinker/models/_models_py3.py:87: SyntaxWarning: invalid escape sequence '\ '
  """
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/monitor/aaz/latest/monitor/action_group/test_notifications/_create.py:18: SyntaxWarning: invalid escape sequence '\ '
  """Create an action group test-notifications.
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/monitor/aaz/latest/monitor/activity_log/alert/_create.py:18: SyntaxWarning: invalid escape sequence '\ '
  """Create a default activity log alert rule.
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/monitor/aaz/latest/monitor/activity_log/alert/_update.py:18: SyntaxWarning: invalid escape sequence '\ '
  """Update a new activity log alert or update an existing one.
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/monitor/aaz/latest/monitor/autoscale/_create.py:15: SyntaxWarning: invalid escape sequence '\ '
  """Create an autoscale setting.
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/monitor/aaz/latest/monitor/autoscale/_update.py:18: SyntaxWarning: invalid escape sequence '\ '
  """Update an autoscale setting.
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/mgmt/resource/resources/v2022_09_01/operations/_operations.py:9171: SyntaxWarning: invalid escape sequence '\ '
  """Get all the resources for a resource group.
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/mgmt/resource/resources/v2022_09_01/operations/_operations.py:9700: SyntaxWarning: invalid escape sequence '\ '
  """Get all the resources in a subscription.
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_create.py:56: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_create.py:186: SyntaxWarning: invalid escape sequence '\.'
  pattern="^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))*$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_create.py:201: SyntaxWarning: invalid escape sequence '\.'
  pattern="^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))*$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_create.py:232: SyntaxWarning: invalid escape sequence '\.'
  pattern="^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))?)?$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_delete.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_renew_credentials.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_show.py:53: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_update.py:59: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_update.py:197: SyntaxWarning: invalid escape sequence '\.'
  pattern="^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))*$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_update.py:215: SyntaxWarning: invalid escape sequence '\.'
  pattern="^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))*$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_update.py:251: SyntaxWarning: invalid escape sequence '\.'
  pattern="^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))?)?$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/_wait.py:49: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_add.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_add.py:104: SyntaxWarning: invalid escape sequence '\.'
  pattern="^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))*$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_add.py:122: SyntaxWarning: invalid escape sequence '\.'
  pattern="^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))*$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_add.py:159: SyntaxWarning: invalid escape sequence '\.'
  pattern="^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))?)?$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_list.py:53: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_remove.py:55: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_show.py:53: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_update.py:56: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_update.py:112: SyntaxWarning: invalid escape sequence '\.'
  pattern="^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))*$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_update.py:133: SyntaxWarning: invalid escape sequence '\.'
  pattern="^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))*$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_update.py:175: SyntaxWarning: invalid escape sequence '\.'
  pattern="^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((, ?)(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))?)?$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/account/ad/_wait.py:49: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_break_file_locks.py:55: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_break_file_locks.py:64: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_break_file_locks.py:78: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_break_file_locks.py:92: SyntaxWarning: invalid escape sequence '\.'
  pattern="^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_create.py:61: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_create.py:69: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_create.py:82: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_create.py:287: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-]{0,79}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_delete.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_delete.py:63: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_delete.py:77: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_finalize_relocation.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_finalize_relocation.py:63: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_finalize_relocation.py:77: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_get_groupid_list_for_ldapuser.py:56: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_get_groupid_list_for_ldapuser.py:65: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_get_groupid_list_for_ldapuser.py:79: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_list.py:53: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_list.py:61: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_migrate_backup.py:52: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_migrate_backup.py:61: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_migrate_backup.py:75: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_pool_change.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_pool_change.py:63: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_pool_change.py:77: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_populate_availability_zone.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_populate_availability_zone.py:63: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_populate_availability_zone.py:77: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_relocate.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_relocate.py:63: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_relocate.py:77: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_reset_cifs_pw.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_reset_cifs_pw.py:63: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_reset_cifs_pw.py:77: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_revert.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_revert.py:63: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_revert.py:77: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_revert_relocation.py:55: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_revert_relocation.py:64: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_revert_relocation.py:78: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_show.py:53: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_show.py:62: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_show.py:76: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_update.py:56: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_update.py:65: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_update.py:79: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_update.py:276: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-]{0,79}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_wait.py:49: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_wait.py:58: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/_wait.py:72: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_create.py:56: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_create.py:67: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_create.py:176: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-]{0,79}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_delete.py:55: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_delete.py:67: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_list.py:52: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_show.py:53: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_show.py:65: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_update.py:53: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_update.py:65: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_update.py:168: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-]{0,79}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_wait.py:49: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume_group/_wait.py:61: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_add.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_add.py:62: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_add.py:75: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_list.py:53: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_list.py:61: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_list.py:74: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_remove.py:55: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_remove.py:63: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_remove.py:76: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_show.py:53: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_show.py:61: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_show.py:74: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_update.py:56: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_update.py:64: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_update.py:77: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_wait.py:49: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_wait.py:58: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/export_policy/_wait.py:72: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_approve.py:56: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_approve.py:65: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_approve.py:79: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_list.py:52: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_list.py:60: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_list.py:73: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_re_initialize.py:57: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_re_initialize.py:66: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_re_initialize.py:80: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_reestablish.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_reestablish.py:63: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_reestablish.py:77: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_remove.py:54: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_remove.py:63: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_remove.py:77: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_resume.py:57: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_resume.py:66: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_resume.py:80: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_status.py:55: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_status.py:64: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_status.py:78: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_suspend.py:57: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_suspend.py:66: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/volume/replication/_suspend.py:80: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/pool/_create.py:55: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/pool/_create.py:63: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/pool/_delete.py:56: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/pool/_delete.py:65: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/pool/_list.py:55: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/pool/_show.py:55: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/pool/_show.py:64: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/pool/_update.py:58: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/pool/_update.py:67: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/pool/_wait.py:49: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,127}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/netappfiles/aaz/latest/netappfiles/pool/_wait.py:58: SyntaxWarning: invalid escape sequence '\-'
  pattern="^[a-zA-Z0-9][a-zA-Z0-9\-_]{0,63}$",
/home/user/.pyenv/versions/3.12.3/lib/python3.12/site-packages/azure/cli/command_modules/network/aaz/latest/network/lb/rule/_update.py:18: SyntaxWarning: invalid escape sequence '\ '
  """Update a load balancing rule.
```

### Issue script & Debug output

N/A

### Expected behavior

The az-cli should not spew any syntax warnings.

### Environment Summary

azure-cli==2.59.0
Python 3.12.3

### Additional context

These syntax issues have been deprecated in Python for several python releases, 3.12 is showing the issues more explicitly since they are planned to break runtime in the future.

This should be fixed very promptly.

Here is the code that need to be fixed:
```
ruff check ./src --select W605                                                                                     

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:382:80: W605 [*] Invalid escape sequence: `\[`
    |
380 |             )
381 |
382 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find '\[1\]' in ''. Available options: \['d', 'd2', 'i', 'l', 'l2', 's'\]"):
    |                                                                                ^^ W605
383 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
384 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:382:83: W605 [*] Invalid escape sequence: `\]`
    |
380 |             )
381 |
382 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find '\[1\]' in ''. Available options: \['d', 'd2', 'i', 'l', 'l2', 's'\]"):
    |                                                                                   ^^ W605
383 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
384 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:382:113: W605 [*] Invalid escape sequence: `\[`
    |
380 |             )
381 |
382 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find '\[1\]' in ''. Available options: \['d', 'd2', 'i', 'l', 'l2', 's'\]"):
    |                                                                                                                 ^^
 W605
383 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
384 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:382:145: W605 [*] Invalid escape sequence: `\]`
    |
380 |             )
381 |
382 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find '\[1\]' in ''. Available options: \['d', 'd2', 'i', 'l', 'l2', 's'\]"):
    |
                              ^^ W605
383 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
384 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:392:113: W605 [*] Invalid escape sequence: `\[`
    |
390 |             )
391 |
392 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find 'props' in ''. Available options: \['d', 'd2', 'i', 'l', 'l2', 's'\]"):
    |                                                                                                                 ^^
 W605
393 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
394 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:392:145: W605 [*] Invalid escape sequence: `\]`
    |
390 |             )
391 |
392 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find 'props' in ''. Available options: \['d', 'd2', 'i', 'l', 'l2', 's'\]"):
    |
                              ^^ W605
393 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
394 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:402:88: W605 [*] Invalid escape sequence: `\[`
    |
400 |             )
401 |
402 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find 'o' in 'l\[0\]'. Available options: \['p', 's'\]"):
    |                                                                                        ^^ W605
403 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
404 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:402:91: W605 [*] Invalid escape sequence: `\]`
    |
400 |             )
401 |
402 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find 'o' in 'l\[0\]'. Available options: \['p', 's'\]"):
    |                                                                                           ^^ W605
403 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
404 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:402:115: W605 [*] Invalid escape sequence: `\[`
    |
400 |             )
401 |
402 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find 'o' in 'l\[0\]'. Available options: \['p', 's'\]"):
    |
^^ W605
403 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
404 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:402:125: W605 [*] Invalid escape sequence: `\]`
    |
400 |             )
401 |
402 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find 'o' in 'l\[0\]'. Available options: \['p', 's'\]"):
    |
          ^^ W605
403 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
404 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:444:88: W605 [*] Invalid escape sequence: `\[`
    |
442 |             )
443 |
444 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find 't' in 'l\[1\]'. Available options: \['p', 's'\]"):
    |                                                                                        ^^ W605
445 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
446 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:444:91: W605 [*] Invalid escape sequence: `\]`
    |
442 |             )
443 |
444 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find 't' in 'l\[1\]'. Available options: \['p', 's'\]"):
    |                                                                                           ^^ W605
445 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
446 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:444:115: W605 [*] Invalid escape sequence: `\[`
    |
442 |             )
443 |
444 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find 't' in 'l\[1\]'. Available options: \['p', 's'\]"):
    |
^^ W605
445 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
446 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:444:125: W605 [*] Invalid escape sequence: `\]`
    |
442 |             )
443 |
444 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find 't' in 'l\[1\]'. Available options: \['p', 's'\]"):
    |
          ^^ W605
445 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
446 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:464:93: W605 [*] Invalid escape sequence: `\(`
    |
462 |             )
463 |
464 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Expect <class 'str'>, got 3 \(<class 'int'>\)"):
    |                                                                                             ^^ W605
465 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
466 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:464:108: W605 [*] Invalid escape sequence: `\)`
    |
462 |             )
463 |
464 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Expect <class 'str'>, got 3 \(<class 'int'>\)"):
    |                                                                                                            ^^ W605
465 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
466 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:484:80: W605 [*] Invalid escape sequence: `\[`
    |
482 |             )
483 |
484 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find '\[10\]' in 'd'. Available options: \['a', 'b', 'c'\]"):
    |                                                                                ^^ W605
485 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
486 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:484:84: W605 [*] Invalid escape sequence: `\]`
    |
482 |             )
483 |
484 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find '\[10\]' in 'd'. Available options: \['a', 'b', 'c'\]"):
    |                                                                                    ^^ W605
485 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
486 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:484:115: W605 [*] Invalid escape sequence: `\[`
    |
482 |             )
483 |
484 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find '\[10\]' in 'd'. Available options: \['a', 'b', 'c'\]"):
    |
^^ W605
485 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
486 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:484:130: W605 [*] Invalid escape sequence: `\]`
    |
482 |             )
483 |
484 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Couldn't find '\[10\]' in 'd'. Available options: \['a', 'b', 'c'\]"):
    |
               ^^ W605
485 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
486 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:495:93: W605 [*] Invalid escape sequence: `\(`
    |
494 |         # test add
495 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Expect <class 'int'>, got a \(<class 'str'>\)"):
    |                                                                                             ^^ W605
496 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
497 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli-core\azure\cli\core\tests\test_aaz_operation.py:495:108: W605 [*] Invalid escape sequence: `\)`
    |
494 |         # test add
495 |         with self.assertRaisesRegex(InvalidArgumentValueError, "Expect <class 'int'>, got a \(<class 'str'>\)"):
    |                                                                                                            ^^ W605
496 |             AAZGenericInstanceUpdateOperation(ctx)._update_instance_by_generic(
497 |                 instance,
    |
    = help: Use a raw string literal

src\azure-cli\azure\cli\__main__.py:11:8: W605 [*] Invalid escape sequence: `\ `
   |
 9 | start_time = timeit.default_timer()
10 |
11 | print('\ ')
   |        ^^ W605
12 |
13 | # import warnings
   |
   = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\monitor\tests\latest\test_monitor_metric_alert_scenarios.py:602:55: W605 [*] Invalid escape sequence: `\%`
    |
600 |         self.cmd(
601 |             'monitor metrics alert create -g {rg} -n {alert2} --scopes {sa_id} --region westus --description "Test"'
602 |             ' --condition "avg My-Ns.\\LogicalDisk(C:)\% Free Space = 1 with skipMetricValidation"',
    |                                                       ^^ W605
603 |             checks=[
604 |                 self.check('description', 'Test'),
    |
    = help: Add backslash to escape sequence

src\azure-cli\azure\cli\command_modules\monitor\tests\latest\test_monitor_metric_alert_scenarios.py:607:78: W605 [*] Invalid escape sequence: `\%`
    |
605 |                 self.check('length(criteria.allOf)', 1),
606 |                 self.check('criteria.allOf[0].metricNamespace', 'My-Ns'),
607 |                 self.check('criteria.allOf[0].metricName', '\\LogicalDisk(C:)\% Free Space'),
    |                                                                              ^^ W605
608 |                 self.check('criteria.allOf[0].skipMetricValidation', True)
609 |             ])
    |
    = help: Add backslash to escape sequence

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2196:192: W605 [*] Invalid escape sequence: `\/`
     |
2194 |         # manage rewrite rules with response-header-configurations
2195 |         self.cmd('network application-gateway rewrite-rule create -g {rg} --gateway-name {gw} --rule-set-name {set} -n {rule4} '
2196 |                  '--sequence 123 --response-header-configurations [{{"header-name":Set-Cookie,"header-value":hat1,"header-value-matcher":{{"ignore-case":true,"negate":true,"pattern":"(https?)\/\/.*azurewebsites\.net(.*)$"}}}}] --request-headers foo=bar '
     |
                                                                              ^^ W605
2197 |                  '--modified-path "/def" --modified-query-string "a=b&c=d%20f"',
2198 |                  checks=[
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2196:194: W605 [*] Invalid escape sequence: `\/`
     |
2194 |         # manage rewrite rules with response-header-configurations
2195 |         self.cmd('network application-gateway rewrite-rule create -g {rg} --gateway-name {gw} --rule-set-name {set} -n {rule4} '
2196 |                  '--sequence 123 --response-header-configurations [{{"header-name":Set-Cookie,"header-value":hat1,"header-value-matcher":{{"ignore-case":true,"negate":true,"pattern":"(https?)\/\/.*azurewebsites\.net(.*)$"}}}}] --request-headers foo=bar '
     |
                                                                                ^^ W605
2197 |                  '--modified-path "/def" --modified-query-string "a=b&c=d%20f"',
2198 |                  checks=[
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2196:211: W605 [*] Invalid escape sequence: `\.`
     |
2194 |         # manage rewrite rules with response-header-configurations
2195 |         self.cmd('network application-gateway rewrite-rule create -g {rg} --gateway-name {gw} --rule-set-name {set} -n {rule4} '
2196 |                  '--sequence 123 --response-header-configurations [{{"header-name":Set-Cookie,"header-value":hat1,"header-value-matcher":{{"ignore-case":true,"negate":true,"pattern":"(https?)\/\/.*azurewebsites\.net(.*)$"}}}}] --request-headers foo=bar '
     |
                                                                                                 ^^ W605
2197 |                  '--modified-path "/def" --modified-query-string "a=b&c=d%20f"',
2198 |                  checks=[
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2203:114: W605 [*] Invalid escape sequence: `\/`
     |
2201 |                      self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.ignoreCase', True),
2202 |                      self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.negate', True),
2203 |                      self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.pattern', "(https?)\/\/.*azurewebsites\.net(.*)$"),
     |
^^ W605
2204 |                  ])
2205 |         self.cmd(
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2203:116: W605 [*] Invalid escape sequence: `\/`
     |
2201 |                      self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.ignoreCase', True),
2202 |                      self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.negate', True),
2203 |                      self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.pattern', "(https?)\/\/.*azurewebsites\.net(.*)$"),
     |
  ^^ W605
2204 |                  ])
2205 |         self.cmd(
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2203:133: W605 [*] Invalid escape sequence: `\.`
     |
2201 |                      self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.ignoreCase', True),
2202 |                      self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.negate', True),
2203 |                      self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.pattern', "(https?)\/\/.*azurewebsites\.net(.*)$"),
     |
                   ^^ W605
2204 |                  ])
2205 |         self.cmd(
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2207:188: W605 [*] Invalid escape sequence: `\/`
     |
2205 |         self.cmd(
2206 |             'network application-gateway rewrite-rule update -g {rg} --gateway-name {gw} --rule-set-name {set} -n {rule4} '
2207 |             '--sequence 123 --response-header-configurations [{{"header-name":Set-Cookie,"header-value":hat2,"header-value-matcher":{{"ignore-case":true,"negate":false,"pattern":"(https?)\/\/.*azurewebsites\.net(.*)$"}}}}] --request-headers foo=bar '
     |
                                                                          ^^ W605
2208 |             '--modified-path "/def" --modified-query-string "a=b&c=d%20f"',
2209 |             checks=[
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2207:190: W605 [*] Invalid escape sequence: `\/`
     |
2205 |         self.cmd(
2206 |             'network application-gateway rewrite-rule update -g {rg} --gateway-name {gw} --rule-set-name {set} -n {rule4} '
2207 |             '--sequence 123 --response-header-configurations [{{"header-name":Set-Cookie,"header-value":hat2,"header-value-matcher":{{"ignore-case":true,"negate":false,"pattern":"(https?)\/\/.*azurewebsites\.net(.*)$"}}}}] --request-headers foo=bar '
     |
                                                                            ^^ W605
2208 |             '--modified-path "/def" --modified-query-string "a=b&c=d%20f"',
2209 |             checks=[
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2207:207: W605 [*] Invalid escape sequence: `\.`
     |
2205 |         self.cmd(
2206 |             'network application-gateway rewrite-rule update -g {rg} --gateway-name {gw} --rule-set-name {set} -n {rule4} '
2207 |             '--sequence 123 --response-header-configurations [{{"header-name":Set-Cookie,"header-value":hat2,"header-value-matcher":{{"ignore-case":true,"negate":false,"pattern":"(https?)\/\/.*azurewebsites\.net(.*)$"}}}}] --request-headers foo=bar '
     |
                                                                                             ^^ W605
2208 |             '--modified-path "/def" --modified-query-string "a=b&c=d%20f"',
2209 |             checks=[
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2215:37: W605 [*] Invalid escape sequence: `\/`
     |
2213 |                 self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.negate', False),
2214 |                 self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.pattern',
2215 |                            "(https?)\/\/.*azurewebsites\.net(.*)$"),
     |                                     ^^ W605
2216 |             ])
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2215:39: W605 [*] Invalid escape sequence: `\/`
     |
2213 |                 self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.negate', False),
2214 |                 self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.pattern',
2215 |                            "(https?)\/\/.*azurewebsites\.net(.*)$"),
     |                                       ^^ W605
2216 |             ])
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\network\tests\latest\test_network_commands.py:2215:56: W605 [*] Invalid escape sequence: `\.`
     |
2213 |                 self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.negate', False),
2214 |                 self.check('actionSet.responseHeaderConfigurations[0].headerValueMatcher.pattern',
2215 |                            "(https?)\/\/.*azurewebsites\.net(.*)$"),
     |                                                        ^^ W605
2216 |             ])
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\storage\tests\latest\test_storage_batch_operations.py:170:49: W605 [*] Invalid escape sequence: `\/`
    |
168 |         if result and result[0]:
169 |             res = result[0]
170 |             self.assertRegex(res['Blob'], '^.*[^\/]+$')
    |                                                 ^^ W605
171 |             base_url = res['Blob'].split('/')[:3]
172 |             container = res['Blob'].split('/')[3]
    |
    = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\vm\tests\latest\test_vm_commands.py:4725:143: W605 [*] Invalid escape sequence: `\.`
     |
4723 |         ])
4724 |         from azure.core.exceptions import HttpResponseError
4725 |         with self.assertRaisesRegex(HttpResponseError, 'Managed Images are not supported for virtual machines with security type TrustedLaunch\.'):
     |
                             ^^ W605
4726 |             self.cmd('vmss update -g {rg} -n {vmss} --security-type TrustedLaunch')
     |
     = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\vm\tests\latest\test_vm_commands.py:11423:54: W605 [*] Invalid escape sequence: `\d`
      |
11421 |         # show snapshot B, check completionPercent
11422 |         self.cmd('snapshot show -g {rg2} -n snapb', checks=[
11423 |             self.check_pattern('completionPercent', '\d?.\d?')
      |                                                      ^^ W605
11424 |         ])
      |
      = help: Use a raw string literal

src\azure-cli\azure\cli\command_modules\vm\tests\latest\test_vm_commands.py:11423:58: W605 [*] Invalid escape sequence: `\d`
      |
11421 |         # show snapshot B, check completionPercent
11422 |         self.cmd('snapshot show -g {rg2} -n snapb', checks=[
11423 |             self.check_pattern('completionPercent', '\d?.\d?')
      |                                                          ^^ W605
11424 |         ])
      |
      = help: Use a raw string literal

Found 41 errors.
[*] 41 fixable with the `--fix` option.
```

--- a/src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_batch_operations.py
+++ b/src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_batch_operations.py
@@ -167,7 +167,7 @@ class StorageBatchOperationScenarios(StorageScenarioMixin, LiveScenarioTest):
                          test_dir, container).get_output_in_json()
         if result and result[0]:
             res = result[0]
-            self.assertRegex(res['Blob'], '^.*[^\/]+$')
+            self.assertRegex(res['Blob'], '^.*[^/]+$')
             base_url = res['Blob'].split('/')[:3]
             container = res['Blob'].split('/')[3]
             blob_name = '/'.join(res['Blob'].split('/')[4:])

